apps/voting/contracts/Voting.sol

```diff
@@ -77,8 +79,12 @@ contract Voting is IForwarder, AragonApp {
     * @param _supportRequiredPct Percentage of yeas in casted votes for a vote to succeed (expressed as a percentage of 10^18; eg. 10^16 = 1%, 10^18 = 100%)
     * @param _minAcceptQuorumPct Percentage of yeas in total possible votes for a vote to succeed (expressed as a percentage of 10^18; eg. 10^16 = 1%, 10^18 = 100%)
     * @param _voteTime Seconds that a vote will be open for token holders to vote (unless enough yeas or nays have been cast to make an early decision)
+    * @param _objectionTime Seconds that a vote will be open for token holders to object to the decision
```

Comment:
@param _objectionTime The duration of the objection vote phase, i.e. seconds that a vote will be open after the main vote phase passes for token holders to object to the outcome